### PR TITLE
add headline

### PR DESCRIPTION
**Pull request by Kevinwey**
Please leave a comment :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,67 @@
...
+  <body>
+    <header>
+      <ul>
+        <li><a href="#Contact">Contact</a></li>
+        <li><a href="#Biography">Biography</a></li>
+      </ul>
+    </header>
+    <main>
+      <h1>Kevin Weyrowitz</h1>
+      <article>
```
Comment (toshydev) [inline index.html]: You could use the `<figure>` element with a nested `<figcaption>` element for this exact purpose.

**Review by toshydev — APPROVED** [review]
**LGTM** :)

**Review by FrankSiyli — APPROVED** [review]
LGTM👌

File: index.html [hunk 2]
```diff
@@ -4,7 +4,67 @@
...
+        <img src="https://bit.ly/3ljrST5" alt="alternatives bild" />
+        <p>This is me in the future doing interesting thinks :)</p>
```
Comment (Satttoshi) [inline index.html]: "thinks" needs to be "things" 🤓

File: index.html [hunk 3]
```diff
@@ -4,7 +4,67 @@
...
+      </article>
+      <hr />
+      <article id="Contact">
+        <p>Num: 0000/0000000</p>
+        <a href="kevinweyrowitz93@gmail.com">kevinweyrowitz93@gmail.com</a>
```
Comment (Satttoshi) [inline index.html]: switch href to href="mailto:kevinweyrowitz93@gmail.com"

**Review by Satttoshi — CHANGES_REQUESTED** [review]
Some small issues, please check

File: index.html (hunk 2 above)
Comment (Kevinwey) [reply index.html]: thx :)

File: index.html (hunk 3 above)
Comment (Kevinwey) [reply index.html]: thx :)

**Review by Kevinwey — COMMENTED** [review]
I will change that :)

File: index.html (hunk 1 above)
Comment (Kevinwey) [reply index.html]: thx for your help :)

**Pull request closed without merging**